### ISSUE TITLE
Is there an older version that supports OpenVPN 2.5?

### ISSUE THREAD

**Issue by mreymann** [jkroepke/openvpn-auth-oauth2#336]
### Problem Statement

I use openvpn-2.5.11-1.el9.x86_64 and openvpn-auth-oauth2-1.22.2-1.x86_64 on AlmaLinux 9.4.

[Your article on medium.com](https://medium.com/@jkroepke/setup-sso-for-openvpn-server-with-microsoft-entra-id-using-oidc-f0d5e103125e) mentions "OpenVPN Server with version 2.5 or above" as a pre-requistite, so I naively expected this to work (which it didn't, see log below).

I see that OpenVPN 2.6.2 or higher is now a requirement. I'd rather avoid installing OpenVPN 2.6 from external repos, so I'd like to know if there is an older version of openvpn-auth-oauth2 that still supports OpenVPN 2.5.

Best regards,
Marc

### openvpn-auth-oauth2 logs

```shell
Okt 30 11:54:29 lb01.example.com openvpn-auth-oauth2[169904]: time=2024-10-30T11:54:29.116+01:00 level=ERROR msg="OpenVPN: OpenVPN management error: openvpn-auth-oauth2 requires OpenVPN management interface version 5 or higher"
```

### Environment

- openvpn-auth-oauth2 Version: openvpn-auth-oauth2-1.22.2-1.x86_64 
- OpenVPN Server Version: openvpn-2.5.11-1.el9.x86_64
- Server OS: AlmaLinux 9.4
- OpenVPN Client (flavor, OS): OpenVPN 2.6.6 on Windows 10 Pro

**Comment by jkroepke** [comment]
The was an initial pre-released version might be run under 2.5. But at some point, there was an critical bug related to WebAuth feature in OpenVPN which was fixed in 2.6.2. To resolve that issue, a breaking change was necessary at OpenVPN side and I was not open to support both. The WebAuth authentication does not work flawless at OpenVPN 2.5, it might be pointless to offer an version which supports 2.5.

The only exception here would be if EPEL would backport that critical issue, but I due the breaking change on the management interface, it will be never happen.

In conclusion, there is may an very old version, but it will have bugs. You will not happy what that state.

I have edit the medium blog to prevent wrong expectations.

In case you have to connect against Azure AD, this might be work under OpenVPN 2.5: https://github.com/jkroepke/openvpn-auth-azure-ad

It did not depend against the WebAuth feature, but there is no support anymore.
